nix/process-compose/default.nix

```
@@ -37,21 +37,14 @@ in
 
   config.outputs =
     let
-      mkProcessComposeWrapper = { name, tui, httpServer, configFile, preHook, postHook }:
+      mkProcessComposeWrapper = { name, cli, preHook, postHook, }:
         pkgs.writeShellApplication {
           inherit name;
           runtimeInputs = [ config.package ];
           text = ''
-            export PC_CONFIG_FILES=${configFile}
-            ${
-              # Once the following issue is fixed we should be able to simply do:
-              # export PC_DISABLE_TUI=${builtins.toJSON (!config.tui)}
-              # https://github.com/F1bonacc1/process-compose/issues/75
-              if tui then "" else "export PC_DISABLE_TUI=true"
-            }
             ${preHook}
 
-            set -x; process-compose ${httpServer.outputs.cliOpts} "$@"; set +x
+            set -x; process-compose ${cli.global.output} ${cli.up.output} "$@"; set +x
```

Shouldn't there be an `up` subcommand here? i.e.,

```suggestion
            set -x; process-compose ${cli.global.output} up ${cli.up.output} "$@"; set +x
```

---

Yup, but "up"'s the default if no subcommand is specified. It was already like this before so I didn't change it.

Additionally, i want to add support for other subcommands in the next PR, and adding "up" there wouldn't make that easier.